src/stream/src/executor/backfill/cdc/cdc_backfill.rs

```diff
@@ -149,6 +212,21 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
         let schema_table_name = self.external_table.schema_table_name().clone();
         let external_database_name = self.external_table.database_name().to_owned();
 
+        // Start background task to periodically query confirm_flush_lsn for PostgreSQL CDC
+        let streaming_metrics = self.streaming_metrics.clone();
+        let source_id = self.actor_ctx.id;
+        let slot_name = self.properties.get("slot.name").cloned();
+        let external_table = self.external_table.clone();
+
+        if let Some(slot_name) = slot_name {
```

Comment:
This will spawn a lsn monitor for every backfill actor.
I believe one lsn monitor for one cdc source is enough.